contracts/rewards/src/msg.rs

```diff
@@ -12,16 +12,16 @@ pub struct InstantiateMsg {
 pub struct RewardsParams {
     /// How often rewards are calculated, specified in number of blocks. Participation is calculated over this window. So if epoch_duration is 500
     /// blocks, workers are rewarded for their participation within each 500 block window.
-    epoch_duration: nonempty::Uint64,
+    pub epoch_duration: nonempty::Uint64,
 
     /// Total number of tokens distributed as rewards per epoch. Tokens are split equally amongst all participating workers for a given epoch
-    rewards_rate: nonempty::Uint64,
+    pub rewards_rate: nonempty::Uint256,
```

Comment:
why not make it explicit?
```suggestion
    pub rewards_per_epoch: nonempty::Uint256,
```